shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/AbstractPipelineJob.java

```diff
@@ -44,6 +44,9 @@ public abstract class AbstractPipelineJob implements PipelineJob {
     @Setter
     private OneOffJobBootstrap oneOffJobBootstrap;
     
+    @Setter
+    private int shardingItem;
```

Comment:
It's better not add `shardingItem` field in job class, since there might be severl sharding items in a job.